remoulade/state/backends/postgres.py

```diff
@@ -0,0 +1,131 @@
+import datetime
+import sys
+from typing import Any, List, Optional, TypeVar
+
+from pytz import utc
+from sqlalchemy import Column, DateTime, LargeBinary, SmallInteger, String, create_engine, inspect, Float
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from remoulade import Encoder
+from remoulade.state import State, StateBackend
+
+Base: Any = declarative_base()
+
+db_string = "postgresql://remoulade@localhost:5544/remoulade"
+T = TypeVar("T", bound="StoredState")
+
+
+class StoredState(Base):
+
+    __tablename__ = "states"
```

Comment:
we will need to add some indexes so that our query won't be too slow. The most import one is on message_id, but we can also add one for status, actor_name and the 3 datetimes